eras/alonzo/impl/CHANGELOG.md

```diff
@@ -7,6 +7,22 @@
 
 ### `testlib`
 
+* Add `ToExpr` instances for:
+  * `CollectError`
+  * `AlonzoUtxoPredFailure`
+  * `FailureDescription`
+  * `TagMismatchDescription`
+  * `AlonzoUtxosPredFailure`
+  * `AlonzoUtxowPredFailure`
+  * `BinaryData`
+  * `Datum`
+  * `IsValid`
+  * `ScriptPurpose`
+  * `TxOutSource`
+  * `TranslationError`
+  * `Addr28Extra`
+  * `DataHash32`
+  * `RdmrPtr`
```

Comment:
Some of these `ToExpr` have already been added in the previous PR, sorry for the conflicts :(

Reply:
I doubt it'll be hard to merge